src/main/java/cloud/fogbow/ras/Main.java

```diff
@@ -93,6 +103,36 @@ public void run(ApplicationArguments args) {
         }
     }
 
+    private void getStorableObject(Object order) {
+        // new = constructor.call()
+        // for field in fields:
+        //   if field has not @Column:
+        //     new.field = old.field
+        //   else:
+        //     fieldMaxSize = field.getMaxSize()
+        //     new.field = truncate(old.field, fieldMaxSize)
+        // return new;
+    }
+
+    private boolean isSizeViolation(TransactionSystemException e) {
+        Throwable e1 = e.getCause();
+        if (e1 != null && e1 instanceof RollbackException) {
+            Throwable e2 = e1.getCause();
+            if (e2 != null && e2 instanceof ConstraintViolationException) {
+                ConstraintViolationException constraintViolationException = (ConstraintViolationException) e2;
+                Set<ConstraintViolation<?>> constraintViolations = constraintViolationException.getConstraintViolations();
+                if (constraintViolations.iterator().hasNext()) {
+                    ConstraintViolation<?> constraintViolation = constraintViolations.iterator().next();
+                    if (constraintViolation.getMessage().startsWith("size must be between")) {
```

Comment:
Is the message "size must be between" defined by us? If so, this should be defined as a constant in the appropriate class in the constant package, and properly documented in the places where it is used. If not, this should be defined as a constant in this class, and should also be properly documented.

Reply:
It's defined by Spring.

Reply:
Actually, this code is dead, I had forgotten to remove it. The actual code is in FogbowDatabaseService.isSizeViolation(). The message is defined by the JPA, so I created a constant field to hold its value.